core/client/runtime/pom.xml

```diff
@@ -90,6 +90,11 @@
             <configuration>
               <finalName>${project.artifactId}-${project.version}-jar-with-dependencies</finalName>
               <relocations>
+                <relocation>
+                  <pattern>META-INF/native/libnetty_transport_native_epoll_x86_64.so</pattern>
```

Comment:
can you put a javadoc here explaining why we need this relocation and possibly link to some sources if you think that is helpful?

Reply:
Updated, thanks.